client/src/www/ui_components/app-root.js

```diff
@@ -828,6 +694,9 @@ export class AppRoot extends mixinBehaviors(
       return;
     }
     this.set('route.path', '/' + page);
+
+    this.$.drawer.open = false;
+    this.$.contactView.reset();
```

Comment:
Just curious if this `reset()` call was fixing a specific bug?

Reply:
But you can't get out of the contact view without going back, which clears the state? I'm just surprised you ran into this, as that would be a bug I think.

Anyway extra reset doesn't hurt, so not blocking.